### DISCUSSION TITLE
Swiper js don't swipe. Pagination not showing hte right page number

### DISCUSSION THREAD

**Discussion by Hashiken50** [nolimits4web/swiper#6110] [Q&A]
New to the swiper.
![frontend-sidejob-skillcheck-2_PC png (1366×7000)](https://user-images.githubusercontent.com/98672550/193432478-75f0d090-d94a-42f0-be37-27f01feffef8.png)

I want to make slide like above.

But it comes out like below.
![Kirameki-card](https://user-images.githubusercontent.com/98672550/193432496-5e24acd6-63ba-44f3-aff2-269e3afc6fcc.png)

I followed the documentation and some reference website but Pagination doesn't show & work properly. When cursor is moved to either prev or next button and clicked, there is no reaction. Three are 5 slides in total.

Thanks in advance.

Here is my code

HTML:

`<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Kirameki-card</title>
  <link rel="stylesheet" href="css/ress.min.css">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/swiper@8/swiper-bundle.min.css"/> 
  <link rel="stylesheet" href="css/styles.css">
  <script src="https://kit.fontawesome.com/800c1ef51d.js" crossorigin="anonymous </script>
</head> 

<body>

<div class="swiper">
      <div class="swiper-container">
      <div class="swiper-wrapper">
        <div class="swiper-slide">
          <div class="slide_text">
            <div class="num">01</div>
            <div class="text">カードお申し込み</div>
          </div>
        </div>
        <div class="swiper-slide">
          <div class="slide_text">
            <div class="num">02</div>
            <div class="text">入会審査・カード発行</div>
          </div>
        </div>
        <div class="swiper-slide"><div class="slide_text">
            <div class="num">03</div>
            <div class="text">カード到着行</div>
          </div></div>
        <div class="swiper-slide"><div class="slide_text">
            <div class="num">04</div>
            <div class="text">アプリをダウンロード</div>
          </div></div>
        <div class="swiper-slide"><div class="slide_text">
            <div class="num">05</div>
            <div class="text">アプリでカード番号を確認</div>
          </div>
        </div>
      </div>
    <div class="gallery_navigation">
        <div class="swiper-button-prev"><i class="far fa-chevron-left"></i></div>
        <div class="swiper-pagination"></div>
        <div class="swiper-button-next"><i class="far fa-chevron-right"></i></div>
    </div>
    </div>
    </div>
<script src="https://cdn.jsdelivr.net/npm/swiper@8/swiper-bundle.min.js"></script>
<script src="js/main.js"></script>  
</body>
</html>`

CSS:
`.swiper {
  padding-top:calc(10/ 1366 * 100vw);;
}

.swiper-container {
  max-width: calc( 552.14/ 1366 * 100vw);
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 0 16;
  padding-bottom: calc(119/ 1366 * 100vw);
  /*margin: 0 auto 0;*/
  padding-right: calc(6/ 1366 * 100vw);
    padding-left: calc(0/ 1366 * 100vw);
    margin-left: calc(404/ 1366 * 100vw);
}

.slider .swiper-slide {
  height: calc( 240/ 1366 * 100vw);
  background:#fff;
  border-radius:calc( 30/ 1366 * 100vw);
  border: 6px solid #B5E9EE;
  display: flex;
  justify-content: center;
  align-items: center;
  margin-right: calc( 50.5/ 1366 * 100vw);
  box-shadow: calc(8/ 1366 * 100vw) calc(8/ 1366 * 100vw) calc(9/ 1366 * 100vw) rgb(0 0 0 / 16%);
}

.slider::before {
  content: '';
    display: block;
    background: linear-gradient(-176deg, #F6F3DC, #F6F3DC 55.5%, transparent 55.5%);
    width: 100%;
    height: 17%;
    position: absolute;
    left: 0;
    top: calc(-18/ 1366 * 100vw);
}

.slider::after {
    z-index: 1;
    content: '';
    display: block;
    background: linear-gradient(184deg, #fff, #fff 55.5%, transparent 40.5%);
    width: 100%;
    height: 24%;
    position: absolute;
    left: 0;
    bottom: calc( -122/ 1366 * 100vw);
}

.slider h2 {
  padding-top: calc(161/ 1366 * 100vw);
  padding-bottom: calc(43/ 1366 * 100vw);
}

.slide_text {
  display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    padding-bottom: calc(51/ 1366 * 100vw);
    padding-right: calc(6/ 1366 * 100vw);
    gap: calc(40/ 1366 * 100vw);
  
}

.num {
  font-family: NotoSansJP-Black;
  font-size: calc(36/ 1366 * 100vw);
  line-height: calc(43.2/ 1366 * 100vw);
  color: #1F2774;
  border-bottom: 3px solid #1F2774;
}

.text {
  font-family: NotoSansJP-Regular;
  font-size: calc(36/ 1366 * 100vw);
  line-height: calc(43.2/ 1366 * 100vw);
}

.swiper-pagination{
  position: absolute;
    font-family: NotoSansJP-Bold;
    font-size: calc( 28/ 1366 * 100vw);
    line-height: calc( 40/ 1366 * 100vw);
    top: calc( 37/ 1366 * 100vw);
    text-align: center;
    display: block;
}

.swiper-button-prev {
  width: calc( 60/ 1366 * 100vw);
  height: calc( 61/ 1366 * 100vw);
  background-color: #F6F3DC;
  border-radius: calc( 100/ 1366 * 100vw);
  color: transparent;
}

.swiper-button-next {
  width: calc( 60/ 1366 * 100vw);
  height: calc( 61/ 1366 * 100vw);
  background-color: #F6F3DC;
  border-radius: calc( 100/ 1366 * 100vw);
  color: transparent;
}

.gallery_navigation .fa-chevron-left:before {
    content: "\f053";
    font-size: calc( 28/ 1366 * 100vw);
    position: absolute;
    top: calc( 17/ 1366 * 100vw);
    left: calc( 21/ 1366 * 100vw);
    color: black;
    font-weight: 900;
}

.gallery_navigation .fa-chevron-right:before {
    content: "\f054";
    font-size: calc( 28/ 1366 * 100vw);
    position: absolute;
    top: calc( 17/ 1366 * 100vw);
    left: calc( 21/ 1366 * 100vw);
    color: black;
    font-weight: 900;
}

.gallery_navigation {
  display: flex;
    justify-content: center;
    align-items: center;
    padding: calc( 52/ 1366 * 100vw) 0;
    position: absolute;
    width: calc( 276/ 1366 * 100vw);
    left: calc( 544/ 1366 * 100vw);
    text-align: center;
    top: calc( 254/ 1366 * 100vw);
}`

JS:
`var swiper = new Swiper('.swiper-container .swiper ', {
  loop: true,
  slidesPerView: 1.5,
  spaceBetween: 62.5,
  centeredSlides: true,
  breakpoints: {
    768: {
      slidesPerView: 3,
      spaceBetween: 80,
      centeredSlides: true,
    },
  },
  pagination: {
    el: '.swiper-pagination',
    type: 'fraction',
    formatFractionCurrent: function (n) {
      return '0' + n;
    },
    formatFractionTotal: function (n) {
      return '0' + n;
    },
  },
  navigation: {
    nextEl: '.swiper-button-next',
    prevEl: '.swiper-button-prev',
  },
});`

**Comment by m4lentas** [comment]
try "custom" type, for example
```
pagination: {
     el: ".swiper-pagination",
     type: 'custom',
     renderCustom: function (swiper, current, total) {
           return '0' + current + '/' + '0' + total; 
     }
},
```
